velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -18,16 +18,34 @@
 #include "velox/common/base/Fs.h"
 #include "velox/expression/Expr.h"
 #include "velox/vector/VectorSaver.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
 
 namespace facebook::velox::test {
 
 namespace {
 
-void printRowVector(const RowVectorPtr& rowVector) {
-  LOG(INFO) << "RowVector contents (" << rowVector->type()->toString() << "):";
+// File names used to persist data required for reproducing a failed test case.
+static const char* kInputVectorFileName = "inputVector";
```

Comment:
the recommendation is to create static strings as std::string_view (so it provides a full std::string-like API). This should be described in our coding style doc.